src/citrine/informatics/descriptors.py

```diff
@@ -94,6 +94,33 @@ def __init__(self, key: str, threshold: float = 1.0):
         self.threshold = threshold
 
 
+class MolecularStructureDescriptor(Serializable['MolecularStructureDescriptor'], Descriptor):
+    """
+    [ALPHA] Material descriptor for an organic molecule (accepts SMILES and InChI String values).
+
+    Parameters
+    ----------
+    key: str
+        The column header key corresponding to this descriptor
+
+    """
+
+    key = properties.String('descriptor_key')
+    typ = properties.String('type', default='Organic', deserializable=False)
+
+    def __eq__(self, other):
+        try:
+            attrs = ["key", "typ"]
+            return all([
+                self.__getattribute__(key) == other.__getattribute__(key) for key in attrs
+            ])
+        except Exception:
```

Comment:
Could you be more specific as to the exception? Allowing for any Exception, and not logging it, might cover up unintended developer error in the future

Reply:
Good catch - this was a lazy copy/paste from above. I'll take this opportunity to try and infer the original intent here.